tensorflow/core/kernels/depthwise_conv_op_gpu.cu.cc

```diff
@@ -714,7 +714,7 @@ void LaunchDepthwiseConv2dGPU(const GpuDevice& device,
 #if GOOGLE_CUDA
                                   : device.getNumCudaMultiProcessors();
```

Comment:
definitely could be merged together now as we all use `device.getNumGpuMultiProcessors()`

Reply:
you are right...will combine the two